frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -403,7 +403,8 @@ function ReaderBookmark:onShowBookmark(match_table)
     -- build up item_table
     local item_table = {}
     local is_reverse_sorting = G_reader_settings:nilOrTrue("bookmarks_items_reverse_sorting")
-    local curr_page = tostring(self.ui:getCurrentPage())
+    local curr_page = self:getBookmarkPageString(self.ui.rolling and self.ui.document:getXPointer()
+        or self.ui.paging.current_page)
```

Comment:
(This is just me, but I don't fear adding a lot of indent for this kind of thing:
```lua
    local curr_page = self:getBookmarkPageString(self.ui.rolling and self.ui.document:getXPointer()
                                                                  or self.ui.paging.current_page)
```
In your version, a quick glance (without looking at the parenthesis) would (wrongly) perceive this as:
`curr_page = self:getBookmarkPageString(...) or self.ui.paging.current_page`

Reply:
Some additions about wrapping long lines would be useful here
https://github.com/koreader/koreader-base/wiki/Coding-style#general
How to indent? Where to put `or` and `and` - at the end of the line or at the beginning of the next one?
Is it okay?
```
    local curr_page = self:getBookmarkPageString(
        self.ui.rolling and self.ui.document:getXPointer() or self.ui.paging.current_page)
```

Reply:
I think we have various personal takes on this (discussed in a few occasions in past reviews), so no hard rule.
(I was just mentionning _my_ take in case you'd like it too, but I won't be upset if you don't :)
For this kind of thing, you could also have it on a single very long line (as the 2nd item in your linked coding style section says).
I'd say read around for the various ways past developpers did this for long `(this and that or what)`, and pick the one you prefer/read easier for you if there is one.

(Not fond of your last example, the abrupt `(` at end of line feel strange :)

Reply:
> Some additions about wrapping long lines would be useful here

The only rule is *never* to wrap on account of some kind of stupid artificial restrictions, only because you feel it's clearer to do so.

When in doubt, don't wrap.

Reply:
PS I think your example is confusing unless you at least do this.

```lua
    local curr_page = self:getBookmarkPageString(
        self.ui.rolling and self.ui.document:getXPointer() or self.ui.paging.current_page
    )
```

Split up into multiple lines I would do something more like this:
```lua
    local curr_page = self:getBookmarkPageString(
        self.ui.rolling
        and self.ui.document:getXPointer()
        or self.ui.paging.current_page
    )
```